### ISSUE TITLE
PowerPC gcc >= 8

### ISSUE THREAD

**Issue by d0mm** [compiler-explorer/compiler-explorer#427]
i use compiler explorer a lot with C11 (-x c command). gcc's C11 support is usable since 4.9. is it possible to update PowerPC gcc? By the way: currently it is declared as version 4.8 but reports 4.8.5.

**Comment by Wuerfel21** [comment]
Would be good to have the version of GCC that ships with devkitPPC, which identifies itself as
```
powerpc-eabi-gcc.exe (devkitPPC release 32) 8.1.0
Copyright (C) 2018 Free Software Foundation, Inc.
This is free software; see the source for copying conditions.  There is NO
warranty; not even for MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
```

**Comment by mattgodbolt** [comment]
https://godbolt.org/z/t2Acci is 6.3.0 at least. I'm going to update this to cover powerpc 8.x

**Comment by Wuerfel21** [comment]
But that's only PowerPC64 (no, -m32 doesn't work). The only 32bit PowerPC choice is indeed 4.8.5.

**Comment by dkm** [comment]
We now have power GCC (vanilla, not a vendor specific toolchain) available (11.2, 12.1, 12.2) and we should now keep them up to date.
Closing this, but feel free to re-open if you thing I've missed something!
